extmod/modurandom.c

```diff
@@ -92,6 +92,14 @@ STATIC mp_obj_t mod_urandom_seed(mp_obj_t seed_in) {
 }
 STATIC MP_DEFINE_CONST_FUN_OBJ_1(mod_urandom_seed_obj, mod_urandom_seed);
 
+#ifdef MICROPY_PY_URANDOM_SEED_INIT_FUNC
+STATIC mp_obj_t mod_urandom___init__() {
+    mod_urandom_seed(mp_obj_new_int(MICROPY_PY_URANDOM_SEED_INIT_FUNC));
```

Comment:
For efficiency it might be better to use `MP_OBJ_NEW_SMALL_INT()` rather than `mp_obj_new_int()`, and that shouldn't compromise the randomness.

Reply:
Done in the new commit!